checkita-core/src/main/resources/db/specific/mssql/V1.1__create_job_state.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE "${defaultSchema}"."results_job_config"
+(
+    "job_id"            TEXT             NOT NULL,
+    "config"            TEXT             NOT NULL,
```

Comment:
In MSSQL db type for `job_id` has to be `VARCHAR(512)` and the one for `config` column has to be `VARCHAR(MAX)`. 

Also need to change `TIMESTAMP` to `DATETIME`